scripts/upgrade

```diff
@@ -65,15 +70,13 @@ ynh_add_nginx_config
 #=================================================
 # ADD A CONFIGURATION
 #=================================================
-#ynh_script_progression --message="Adding a configuration file..." --weight=1
-
-#email=$(ynh_user_get_info --username=$admin --key=mail)
+ynh_script_progression --message="Upgrading a configuration file..." --weight=1
 
 # Setup application config
-#ynh_add_config --template=".env.example" --destination="$install_dir/.env"
+ynh_add_config --template=".env.example" --destination="$install_dir/.env"
 
-#chmod 644 "$install_dir/.env"
-#chown $app:$app "$install_dir/.env"
+chmod 644 "$install_dir/.env"
```

Comment:
Uuuuuh I'm not super enthusiast about enabling read write for "all users" (644 instead of 640), is there any reason for this ?

I realize it's also set to 644 in the install_dir

Ultimately this shouldnt be a super huge issue because $install_dir shouldnt have `r` nor `x` for "others", but still

Reply:
Okay, I just made a copy/paste from https://github.com/YunoHost-Apps/2FAuth_ynh/blob/77ebeb85ba6c9c7a89a316bc0d65955218a2ba1b/scripts/install#L68.
I think 640 would be enough, I will have a try soon